resources/scripts/download_libs.py

```diff
@@ -181,37 +111,48 @@ def retrieve_libraries_for_specific_version(
     """
     s3 = boto3.client("s3")
     successful = True
+    version = data["version"]
 
     for arch, arch_data in data["architectures"].items():
         logging.info(f" - Downloading minimal package for {arch_data['name']} ...")
+        filename = f"rti_connext_dds-{version}-min-{arch_data['name']}.zip"
         remote_file_path = PurePosixPath(
             storage_path,
             version,
-            f"rti_connext_dds-{version}-min-{arch_data['name']}.zip",
+            filename,
         )
+        download_dest = Path(f"/tmp/connext_downloads/{filename}")
+        download_dest.parent.mkdir(parents=True, exist_ok=True)
 
         try:
-            response = s3.get_object(Bucket=storage_url, Key=str(remote_file_path))
+            with open(download_dest, "wb") as f:
```

Comment:
From now on, the files will be downloaded to /tmp/ instead of downloading them to memory. This is because the memory in the Jenkins agent is very limited.